documentation/asciidoc/computers/config_txt/overclocking.adoc

```diff
@@ -60,7 +60,7 @@ WARNING: Setting any overclocking parameters to values other than those used by
 | Forces turbo mode frequencies even when the ARM cores are not busy. Enabling this may set the warranty bit if `over_voltage_*` is also set.
 
 | initial_turbo
-| Enables https://forums.raspberrypi.com/viewtopic.php?f=29&t=6201&start=425#p180099[turbo mode from boot] for the given value in seconds, or until cpufreq sets a frequency. The maximum value is `60`.
+| Enables https://forums.raspberrypi.com/viewtopic.php?f=29&t=6201&start=425#p180099[turbo mode from boot] for the given value in seconds, or until cpufreq sets a frequency. The maximum value is `60`.  In newer versions of the firmware (rpi-update 2024/11) the default has been changed to `60` in order to reduce boot time. In addition, the Kernel CPU performance governor has been switched from `powersave` to `ondemand`.
```

Comment:
"(rpi-update 2024/11)" -> "(added to `rpi-update` in November 2024)" ?

Also should probably be a lower-case 'k' on kernel?

Reply:
I think 2024/11 is unambiguous

Reply:
```suggestion
| Enables https://forums.raspberrypi.com/viewtopic.php?f=29&t=6201&start=425#p180099[turbo mode from boot] for the given value in seconds, or until `cpufreq` sets a frequency. The maximum value is `60`. The November 2024 firmware update made the following changes:

* changed the default to `60` to reduce boot time
* switched the kernel CPU performance governor from `powersave` to `ondemand`

```

While 2024/11 is unambiguous, I think these changes make the details a bit clearer.
For the first bullet point (the default), it would be nice to know what the _old_ default was for reference!

Reply:
How about from '0' to '60'